src/riak_kv_pb_object.erl

```diff
@@ -369,6 +394,17 @@ process(
             DW = decode_quorum(DW0),
             PW = decode_quorum(PW0),
             NodeConfirms = decode_quorum(NodeConfirms0),
+            SyncOnWrite =
```

Comment:
Should this not be done by a decode function in `riak_pb`? Feels a bit weird to see the decoding happen here.